lane.py

```diff
@@ -1047,6 +1047,14 @@ def search(self, _db, query, search_client, pagination=None):
         # Get the search results from Elasticsearch.
         results = None
 
+        if not media:
+            media = self.media
+        else:
+            media = [media]
+
+        if not languages:
+            languages = self.languages
```

Comment:
Lane languages should take precedence over user-specified languages, not vice versa. If your browser preferences are set to English but you go into a Spanish lane and run a search, you should search the Spanish collection.